.circleci/config.yml

```diff
@@ -41,8 +41,7 @@ step-library:
         name: Prepare
         command: |
           echo "foo" > ~/.mapbox
-          brew unlink carthage
-          brew install carthage --HEAD
+          brew install carthage
```

Comment:
This caused the build bots to fail because Homebrew/homebrew-core#56511 hadn’t landed. The tests should install again after rerunning.

Reply:
[Builds are failing](https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-ios/1036/workflows/65ef27a8-8131-4dfb-9a68-b4bb3bd01067/jobs/16094/steps) because Carthage is already installed; this configuration needs to upgrade Carthage instead:

```
Warning: carthage 0.33.0 is already installed and up-to-date
 
To reinstall 0.33.0, run `brew reinstall carthage`
 ```